frontend/src/Components/MainContainer/MainContainer.module.css

```diff
@@ -89,6 +91,43 @@
   margin-top: 23px;
 }
 
+@media screen and (max-width: 1280px) and (max-height: 720px) {
+  .paragraph {
+    font-size: 28px;
+  }
+
+  .title {
+    font-size: 60px;
+  }
+
+  .mainContainer {
+    
+    height: 100vh;
+  }
+
+  .questionsContainer {
+    padding: 0%;
+
+  }
+  .category{
+    font-size: 1.3rem;
+
+  }
+
+  .playerContainer{
+    margin-top: 1rem;
+  }
+
+}
+
+
```

Comment:
Could you please check your formatter? Your commits have so much empty lines in them. It is not a big deal, but it hurts the readability.